b2sdk/transfer/inbound/file_metadata.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         self.file_name = file_name
         self.content_type = content_type
         self.content_length = content_length
-        self.content_sha1 = content_sha1
+        self.content_sha1 = self._parse_content_sha1(content_sha1)
```

Comment:
should FileMetadata get a new field that tells whether the checksum we are dealing with here is verified or not? Seems easy to add, it's small but can be important for some callers.

Reply:
Again, it's not part of API so there is no such caller yet.

Reply:
Added